indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -261,7 +261,9 @@
         platforms (map util/map-keys->kebab-case platforms)
         kms-index (kf/get-kms-index context)
         platforms-nested (map #(platform/platform-short-name->elastic-doc kms-index %)
```

Comment:
No need to populate this field value now as it is no longer used by search. Any derived field values (e.g. platform-short-names, etc.) should be derived from platforms2-nested.